debugger_protocol.rst

```diff
@@ -1100,6 +1102,11 @@ a *function* breakpoint.
 This command is used by the IDE to update one or more attributes of a
 breakpoint that was already set on the debugger engine via *breakpoint_set*.
 
+If a debugger engine supports the resolved state for breakpoints, the resolved
```

Comment:
In https://github.com/derickr/dbgp/pull/8/ it looks like the types for the resolved attribute are "RESOLVED" or "UNRESOLVED" but here you say "0".  Is "0" a third state?  Also if we're at the point where we have the "resolved" flag, doesn't that mean we've sent the breakpoint_get or breakpoint_list command, and if so, how do we then "run rediscovery"?  I thought rediscovery was done through those commands.